OpenEdXMobile/src/main/java/org/edx/mobile/http/notifications/EdxErrorState.kt

```diff
@@ -27,7 +27,7 @@ class EdxErrorState @JvmOverloads constructor(
         }
     }
 
-    fun setState(state: State) {
+    fun setState(state: State, isMyCoursesScreen: Boolean = false) {
```

Comment:
better to define some screen enums, and constants. OR we can use `org.edx.mobile.deeplink.Screen`